### PR TITLE
Implment DB setup_3rd attempt

### PR DESCRIPTION
**Pull request by blinker1104**
Use sequelize to sync with PostgreSQL DB server.
Need User account with DBCREATE privilege
Check the dbname, username, and password before running.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,10 +8,20 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "dbSetup": "node ./server/database/db.js"
   },
   "dependencies": {
+    "pg": "^8.2.1",
+    "pg-hstore": "^2.3.3",
+    "postgres": "^1.0.2",
```
Comment (nathanksou) [inline package.json:16]: Uninstall necessary dependencies.

File: package.json [hunk 2]
```diff
@@ -8,10 +8,20 @@
...
+    "sequelize": "^5.21.13"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "babel-loader": "^8.1.0",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "faker": "^4.1.0",
+    "webpack": "^4.43.0"
```
Comment (nathanksou) [inline package.json:25]: Test

File: package.json (hunk 1 above)
Comment (nathanksou) [inline package.json:16]: Uninstall unnecessary dependencies.

File: server/database/db.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+// Option 1: Passing parameters separately
+const sequelize = new Sequelize('mydb', 'noobdev', 'post', {
+  host: 'localhost',
+  dialect: 'postgres'
+});
+
+
+var biz = sequelize.define('business', {
+  // "_id" : Sequelize.INTEGER,
```
Comment (nathanksou) [inline server/database/db.js:12]: What's this line for and why is it uncommented?

**Comment by blinker1104** [comment]
Thank you Nathan

**Pull request merged**